cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -820,7 +841,21 @@ def test_service_dialog_expression_method():
             4.
             5. Expression method should work
     """
-    pass
+    sd, ele_label = import_dialog
+    catalog_item = appliance.collections.catalog_items.create(
+        appliance.collections.catalog_items.GENERIC,
+        name=fauxfactory.gen_alphanumeric(),
+        description=fauxfactory.gen_alphanumeric(),
+        display_in=True,
+        catalog=catalog,
+        dialog=sd,
+    )
+    request.addfinalizer(catalog_item.delete_if_exists)
+    service_catalogs = ServiceCatalogs(
+        appliance, catalog=catalog_item.catalog, name=catalog_item.name
+    )
+    view = navigate_to(service_catalogs, "Order")
+    assert full_vm.name in [opt.text for opt in view.fields("dropdown_list_1").dropdown.all_options]
```

Comment:
Q: is it ok to use `has_item` of `Dropdown` widget to check vm name?

Reply:
@digitronik 
This dropdown is of widget type ```BootstrapSelect```. Hence I am getting below error:
```ipdb> view.fields("dropdown_list_1").dropdown.has_item(full_template_vm.name)                                                                                                
*** AttributeError: 'BootstrapSelect' object has no attribute 'has_item'
```